app/components/certificate/certificate-display.tsx

```diff
@@ -0,0 +1,111 @@
+import {
+  Flex,
+  Box,
+  HStack,
+  Text,
+  Heading,
+  Tooltip,
+  IconButton,
+  Accordion,
+  AccordionItem,
+  AccordionButton,
+  AccordionIcon,
+  AccordionPanel,
+  useToast,
+} from '@chakra-ui/react';
+import { DownloadIcon, CopyIcon } from '@chakra-ui/icons';
+import type { Certificate } from '@prisma/client';
+
+interface Props {
```

Comment:
I started using naming for props as name of component + Props, so I think we should stick to start naming, so it would be like `CertificateDisplayProps`

Reply:
you got a point